app/src/main/java/de/tum/in/tumcampusapp/component/ui/overview/InformationActivity.kt

```diff
@@ -55,16 +56,24 @@ class InformationActivity : BaseActivity(R.layout.activity_information) {
         }
     }
 
+    /**
+     * Open the Facebook app or view page in a browser if Facebook is not installed
+     */
     private fun openFacebook() {
-        // Open the Facebook app or view page in a browser if Facebook is not installed
+        val default = Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.facebook_link)))
         val facebook = try {
             packageManager.getPackageInfo("com.facebook.katana", 0)
             Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.facebook_link_app)))
-        } catch (e: PackageManager.NameNotFoundException) {
-            Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.facebook_link)))
+        } catch (e: Exception) {
+            default
         }
 
-        startActivity(facebook)
+        try {
+            startActivity(facebook)
+        } catch (e: Exception) {
+            // Don't make any assumptions about another app, just start the browser instead
+            startActivity(default)
```

Comment:
Just run it in the catch about. No need to separate it into its own catch, right?